OSSM-8862 Migration Hub page content (#88651)
openshift/openshift-docs: modules/ossm-migrating-hub-premigration-checklists.adoc

// Module included in the following assemblies:
//
// * service-mesh-docs-main/migrating/ossm-migrating-from-2-to-3-assembly.adoc

:_mod-docs-content-type: REFERENCE
[id="ossm-migrating-hub-premigration-checklists_{context}"]
= Premigration checklists

> There are a series of premigration checklists that must be completed before you can begin your migration.

Reviewer: ```suggestion
Before you can begin your migration, complete the premigration checklists.
```
You can engage the reader a bit better by using the imperative mood: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#mood

> There are a series of premigration checklists that must be completed before you can begin migrating your deployment, workloads, and gateways.

Reviewer 2: What does `deployment` mean in this context? Is that your whole mesh?

> These checklists include steps for managing your network policies and configuration updates to set up add ons such as {kialiproduct}, and {temposhortname}.

Reviewer: ```suggestion
These checklists include steps for managing your network policies and configuration updates to set up add ons such as {kialiproduct} and {temposhortname}.
```
